src/test/resources/log/simulation.log

```diff
@@ -1,4 +1,4 @@
-RUN	org.kibanaLoadTest.simulation.DemoJourney	demojourney	1606920741968	 	3.4.2
+RUN	org.kibanaLoadTest.simulation.local.DemoJourney	demojourney	1606920741968	 	3.4.2
```

Comment:
In this case `local` really means `local` and not `branch`?

Reply:
this is actually test. I will change it to `branch` for consistency